core/src/avm2/globals/flash/utils/ByteArray.as

```diff
@@ -2,6 +2,7 @@ package flash.utils {
     [Ruffle(InstanceAllocator)]
     public class ByteArray implements IDataInput2, IDataOutput2 {
         private static var _defaultObjectEncoding:uint = 3;
+        public var shareable:Boolean = false;
```

Comment:
`shareable` should be an accessor, not a variable + it should be marked as stubbed + it should be maked as API(684)